### PR TITLE
Wait for workspace operations during integration test initialization

### PR DESCRIPTION
**Pull request by sharwell**
This change attempts to stabilize some of the flaky integration tests by introducing two wait operations that first appeared to help in #28173.

### CODE REVIEW

File: src/VisualStudio/IntegrationTest/TestUtilities/OutOfProcess/SolutionExplorer_OutOfProc.cs [hunk 1]
```diff
@@ -43,7 +43,10 @@ public void OpenSolution(string path, bool saveExistingSolutionIfExists = false)
             => _inProc.OpenSolution(path, saveExistingSolutionIfExists);
 
         public void AddProject(ProjectUtils.Project projectName, string projectTemplate, string languageName)
-            => _inProc.AddProject(projectName.Name, projectTemplate, languageName);
+        {
+            _inProc.AddProject(projectName.Name, projectTemplate, languageName);
+            _instance.Workspace.WaitForAsyncOperations(FeatureAttribute.Workspace);
```
Comment (jasonmalinowski) [inline src/VisualStudio/IntegrationTest/TestUtilities/OutOfProcess/SolutionExplorer_OutOfProc.cs]: What asynchrony in the workspace is this waiting for?

Comment (sharwell) [reply src/VisualStudio/IntegrationTest/TestUtilities/OutOfProcess/SolutionExplorer_OutOfProc.cs]: I don't remember exactly.

File: src/VisualStudio/IntegrationTest/IntegrationTests/AbstractEditorTest.cs [hunk 2]
```diff
@@ -68,6 +69,7 @@ protected void SetUpEditor(string markupCode)
             VisualStudio.Workspace.SetPrettyListing(LanguageName, false);
             try
             {
+                VisualStudio.Workspace.WaitForAsyncOperations(FeatureAttribute.Workspace);
```
[the remark this answers is not in the document]
Comment (sharwell) [reply src/VisualStudio/IntegrationTest/IntegrationTests/AbstractEditorTest.cs]: Also don't remember exactly on this one 😀

**Comment by sharwell** [comment]
Superseded by #33842

**Pull request closed without merging**